examples/Sniffator/android/app/build.gradle

```diff
@@ -136,13 +147,11 @@ dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
-    compile project(':react-native-ble-plx')
```

Comment:
I think that we need to configure `react-native-ble-plx` once again. It is up to you if you would like to do it in separate commit.

Reply:
Done, thanks for pointing that!